CHANGELOG.rst

```diff
@@ -263,6 +263,16 @@ General
 Fixed
 ~~~~~
 
+:ref:`debops.gitlab_runner` role
+''''''''''''''''''''''''''''''''
+
+- Solve 'Build occasionally get "No such file or directory" accessing build directory' error.
+  See also https://gitlab.com/gitlab-org/gitlab-runner/issues/1379 .
+  Occasionally the runner stops with an error due to the conflicts created by `~/.bash_logout`
+  wipping the screen/terminal clean, thus confusing the runner. Therefore updated the
+  `debops.gitlab_runner` user creation to no add an user skeleton and just an empty home dir.
+  Additionally added task to clean up manually any `~/.bash_logout` files from current installations.
+
```

Comment:
This section should definitely be lower since the roles should be ordered alphabetically. Also check if it's in the "unreleased" section.